server/routes/amendAReferral/amendAReferralController.ts

```diff
@@ -132,4 +134,20 @@ export default class AmendAReferralController {
 
     return ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
+
+  async getChangelog(req: Request, res: Response): Promise<void> {
```

Comment:
changelog part of logic can be moved to Changelog controller as it is not part of the amend feature

Reply:
created the Changelog controller

Reply:
But I think you forgot to remove it from here. 😄